server/base/src/main/java/org/apache/accumulo/server/metadata/ServerAmpleImpl.java

```diff
@@ -170,8 +173,8 @@ public void deleteGcCandidates(DataLevel level, Collection<String> paths) {
       }
       scanner.setRange(range);
 
-      return Iterators.transform(scanner.iterator(), entry -> entry.getKey().getRow().toString()
-          .substring(MetadataSchema.DeletesSection.getRowPrefix().length()));
+      return Iterators.transform(scanner.iterator(),
+          entry -> entry.getKey().getRow().toString().substring(getDeletePrefixLength()));
```

Comment:
This could be `MetadataSchema.DeletesSection.decodeDeleteRow(entry.getKey().getRow())` or similar.

Reply:
resulted in cleaner implemenation. thanks.